### DISCUSSION TITLE
Delay Animation Before Screenshot Start

### DISCUSSION THREAD

**Discussion by putudian2013** [FRSOURCE/cypress-plugin-visual-regression-diff#278] [Q&A]
Hi all,

Is there any way to wait for animation to complete before run the matchImage function? 
Whenever I run the matchImage function it will screenshot the incomplete state of the element.

Thanks,
Dian

**Comment by FRSgit** [comment]
Most common solution is to just disable animation using and old "trick" like this one:

```css
* {
    animation-duration: 0s !important;
}
```

If you inject a css style like above during setup phase of your Cypress test suite - you should be fine :)
